server/test/db.cpp

```diff
@@ -216,15 +216,27 @@ static void createItem(fty::db::Connection& conn, const DBData::Item& item, std:
         }
     }
 
-    for (const auto& tag: item.tags) {
-        if (conn.execute("INSERT INTO t_bios_tag (name) VALUES(:tag)", "tag"_p = tag)) {
+    for (const auto& tag : item.tags) {
+        if (const auto row = conn.select("SELECT * FROM t_bios_tag WHERE name=:tag", "tag"_p = tag);
```

Comment:
const auto row... this const has no sense here. :)

Reply:
And here could be logical error... 
You should clean up you tests and always use fresh data. Otherwise your test become self dependent, so, update of the data is risky.

Reply:
It is needed for db creation, If we create multiple same tags it will return error. This code helps to avoid it